src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -801,6 +792,42 @@ public boolean isKnownParticipant( UUID studyId, String participantId ) {
                 .collect( Collectors.toSet() );
     }
 
+    @Scheduled( fixedRate = 60000 )
+    public void refreshUserAppsDictionary() {
+        SearchApi searchApi;
+        try {
+            ApiClient apiClient = apiClientCache.get( ApiClient.class );
+            searchApi = apiClient.getSearchApi();
+        } catch ( ExecutionException e ) {
+            logger.error( "Unable to load api" );
+            return;
+        }
+
+        logger.info( "Refreshing chronicle user apps dictionary" );
+
+        List<Map<FullQualifiedName, Set<Object>>> searchResult = searchApi
+                .executeEntitySetDataQuery( userAppsDictESID, new SearchTerm( "*", 0, SearchApi.MAX_SEARCH_RESULTS ) )
+                .getHits();
+        logger.info( "fetched {} items from user apps dictionary entity set", searchResult.size() );
+
+        Map<String, String> appsDict = new HashMap<>();
+        searchResult
+                .forEach( entity -> {
+                    String packageName = entity.get( FULL_NAME_FQN ).iterator().next().toString();
+                    String appName = entity.get( TITLE_FQN ).iterator().next().toString();
+                    String recordType = entity.getOrDefault( RECORD_TYPE_FQN, Set.of( "" ) ).iterator().next()
+                            .toString();
+                    if ( !recordType.equals( RecordType.SYSTEM.name() ) ) {
+                        appsDict.put( packageName, appName );
+                    }
```

Comment:
It's good practice to use the enum's `.equals()` since it's guaranteed to not be null:

```
if ( RecordType.SYSTEM.name().equals( recordType ) ) {
  ...
}
```